src/lib/schemeUtils.ts

```diff
@@ -194,7 +194,17 @@ const schemeActionPropNames = new Map<string, Map<GetSchemeIsActiveActions, stri
   ],
 ]);
 
+/**
+ * Returns true or false indicating whether the scheme is active and thus can accept new proposals.
+ * @param scheme Required parameter that if undefined or null will cause this method to throw an exception.
+ * @param action For SchemeRegistrar where we are asking specifically about Add and Remove actions.
+ */
 export function getSchemeIsActive(scheme: ISchemeState, action?: GetSchemeIsActiveActions): boolean {
+
+  if (!scheme) {
+    throw new Error("getSchemeIsActive: scheme parameter is not set");
```

Comment:
does this will crash alchemy ?

Reply:
how about it will just return false here as null scheme is not an active one ?

Reply:
That would be the wrong thing to do.  See https://github.com/daostack/alchemy/pull/1889#discussion_r452294523

Reply:
does this error manifested in the ui somehow or just in the logs ?

Reply:
I do not see a way in which our code, with this PR, can cause this exception to happen.